types/node/net.d.ts

```diff
@@ -73,10 +70,8 @@ declare module "net" {
 
         // Extended base methods
         end(): void;
-        end(buffer: Buffer, cb?: Function): void;
-        end(str: string, cb?: Function): void;
-        end(str: string, encoding?: string, cb?: Function): void;
-        end(data?: any, encoding?: string): void;
+        end(buffer: Buffer | Uint8Array | string, cb?: () => void): void;
```

Comment:
Not directly related but I think `end()` returns `this`.

Reply:
This seems to be the case everywhere actually!

Reply:
Most likely but before there was `any` allowed.

Reply:
Seems this causes a lot other packages to fail so maybe keep `void` for now

Reply:
>.>

Reply:
Actually, let me try fix those 90% of the error seems to be related to `bunyan`